modules/create-issues-in-different-jira-projects.adoc

```diff
@@ -6,18 +6,19 @@
 = Creating issues in different Jira projects
 
 [role="_abstract"]
-You can configure {product-title} to to create issues in different Jira projects so that they directly go to the right team.
+You can configure {product-title} to create issues in different Jira projects so that they directly go to the right team.
 
 .Prerequisites
 * You must have an account with access to each project that you want to send the alerts to.
 
 .Procedure
-. Add an annotation similar to the following in your deployment YAML file:
+. Add an annotation similar to the following example in your namespace or deployment YAML file:
 +
-[source,yaml]
+[source,yaml,subs="+quotes"]
 ----
-jira/project-key: <jira_project_key>
+annotations:
```

Comment:
According to the [OCP guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values): 

![Screenshot from 2024-08-26 12-30-45](https://github.com/user-attachments/assets/fda8b16e-6ff5-42d5-bf3c-d693ab5f0d04)